packages/kbn-optimizer/limits.yml

```diff
@@ -107,7 +107,7 @@ pageLoadAssetSize:
   dataVisualizer: 27530
   banners: 17946
   mapsEms: 26072
-  timelines: 230410
+  timelines: 251886
```

Comment:
I'm talking with @michaelolo24 about this directly, it sounds like this number is going to keep going up unless we get a handle on what's going on here and causing the page load asset size to keep creeping up.